core/templates/pages/topic-viewer-page/info-tab/topic-info-tab.directive.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the topic info tab.
+ */
+
+require('domain/utilities/url-interpolation.service.ts');
+require('services/contextual/window-dimensions.service.ts');
+
+angular.module('oppia').directive('topicInfoTab', ['UrlInterpolationService',
+  function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      bindToController: {
+        getTopicName: '&topicName',
+        getTopicDescription: '&topicDescription',
+        getStoryCount: '&storyCount',
+        getSubtopicCount: '&subtopicCount',
+        getChapterCount: '&chapterCount'
+      },
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/topic-viewer-page/info-tab/topic-info-tab.directive.html'),
+      controllerAs: '$ctrl',
+      controller: [
+        'WindowDimensionsService', function(WindowDimensionsService) {
+          var ctrl = this;
+
+          ctrl.$onInit = function() {
+            ctrl.isMobile = false;
```

Comment:
screenHasSmallWidth might be a better name -- "isMobile" suggests an actual mobile device which may not be the case.

Reply:
Done